### PR TITLE
Added rollbar configuration

### PR DESCRIPTION
**Pull request by balagan73**
#634 

- Added rollbar configuration

### CODE REVIEW

File: web/sites/default/settings.pantheon.php [hunk 1]
```diff
@@ -70,9 +70,12 @@
     }
   }
 }
-
-$pantheon_env = getenv('PANTHEON_ENVIRONMENT');
 if (!empty($pantheon_env)) {
+  // Rollbar settings for LIVE and TEST.
+  if ($pantheon_env == 'live' || $pantheon_env == 'test') {
+    $config['rollbar.environment'] = $pantheon_site_name . '.' . $pantheon_env;
+    $config['rollbar.enabled'] = TRUE;
```
Comment (amitaibu) [inline web/sites/default/settings.pantheon.php:77]: Here we would also have the access token of rollbar right? If so, let's add a placeholder

Comment (balagan73) [reply web/sites/default/settings.pantheon.php:77]: No. Access token is saved in the module's config file, but there is a setting which disables the module, here we just enable it.

**Review by amitaibu — COMMENTED** [review]
Let's add a note in the README, about settings Rollbar.

Maybe under `## Prepare for Deploy` section or similar

File: README.md [hunk 2]
```diff
@@ -215,6 +215,10 @@ if you are using other IDE or want to know more about this feature.
 
 ## Deploy to Pantheon
 
+### Rollbar configuration
+
+Rollbar will be automatically enabled on test and live environments. The access tokens need to be added under '/admin/config/services/rollbar' and the rollbar configuration yml file should be committed.
```
Comment (amitaibu) [inline README.md:220]: Actually, instead of `/admin/config/services/rollbar` - can't we have the setting ready on `settings.pantheon.php`?

**Review by AronNovak — APPROVED** [review]

**Pull request merged**